website/docs/r/user.html.markdown

```diff
@@ -69,7 +69,7 @@ The following arguments are supported:
 * `data_manage_datafeeds` - (Optional) Whether the user can modify data feeds.
 * `account_manage_users` - (Optional) Whether the user can modify account users.
 * `account_manage_payment_methods` - (Optional) Whether the user can modify account payment methods.
-* `account_manage_plan` - (Optional) Whether the user can modify the account plan.
+* `account_manage_plan` - (This is now obsolete.)
```

Comment:
If this option is no longer supported we should formally deprecate it in the provider to let people know it will be removed in the next major version.  As such I think this should look more like:
```
(Optional) **Deprecated** Whether the user can modify the account plan.
```